pkg/sentry/arch/arch_amd64.go

```diff
@@ -12,8 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build amd64
```

Comment:
I know these are actually redundant, but we tend to leave them as a clue to the reader.

Reply:
OK.